### PR TITLE
Added Eddiejaoude and Crossroads

### PR DESCRIPTION
**Pull request by FayasNoushad**

### CODE REVIEW

File: YouTubeChannels.md [hunk 1]
```diff
@@ -69,6 +69,10 @@
 
 * [Google Developers](https://www.youtube.com/user/GoogleDevelopers): The Google Developers channel offers some lessons, tutorials, and guides on various development topics. However, the most focus is on product reviews, recorded talks from tech experts at Google events, and current industry news that are really important for developers.
 
+* [Eddie Joude](https://www.youtube.com/user/eddiejoude): Do you want to get into coding, the community and open source? Do you want to make more money, have flexible and remote working? Subscribe now to my channel and watch my videos to learn how.
```
Comment (AvidCoder101) [inline YouTubeChannels.md:72]: @FayasNoushad, could you please correct Eddie's surname to `Jaoude` in both the array brackets and the url please?

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]

File: YouTubeChannels.md (hunk 1 above)
Comment (FayasNoushad) [reply YouTubeChannels.md]: Ohh! Sorry my mistake. Its fixed now

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]
LGTM

**Comment by AvidCoder101** [comment]
sorry, pressed the wrong one

**Review by AvidCoder101 — APPROVED** [review]
LGTM

**Pull request merged**